### DISCUSSION TITLE
Assembly page: zoomable images

### DISCUSSION THREAD

**Discussion by MichaelMathieu** [upkie/upkie#181] [Ideas]
It would be nice if the images on the assembly page would be zoomable either by clicking or hovering with the mouse.

**Comment by stephane-caron** [comment]
GitHub formatting puts some restrictions, but we can make images clickable with hyperlinks :+1: All pictures in the build instructions now have links, so *e.g.* middle-clicking on an image should open it in full in a separate tab.
